lib/stimulus_reflex/channel.rb

```diff
@@ -1,6 +1,18 @@
 # frozen_string_literal: true
 
-class StimulusReflex::Channel < ActionCable::Channel::Base
+module ApplicationCable
+  class Channel < ActionCable::Channel::Base
+    def initialize(connection, identifier, params = {})
+      super
+      application_cable = Rails.root.join("app", "channels", "application_cable", "channel.rb")
```

Comment:
I’d maybe rename this to `application_channel`, since you’re looking up `application_cable/channel.rb`, right? Might be somewhat ambiguous otherwise...